tools/isobuild/builder.js

```diff
@@ -687,7 +687,16 @@ Previous builder: ${previousBuilder.outputPath}, this builder: ${outputPath}`
     // case of renameDirAlmostAtomically since that one is constructing files to
     // be checked in to version control, but here we could get away with it.
     if (this.buildPath !== this.outputPath) {
-      files.renameDirAlmostAtomically(this.buildPath, this.outputPath);
+      try {
+        files.rename(this.buildPath, this.outputPath);
```

Comment:
This portion was just a proposal.  This is likely failing because the `rename`s and `rm_recursive`s from `renameDirAlmostAtomically` are necessary to make way for the incoming tree.

As a thought, maybe we should continue to use `renameDirAlmostAtomically` here instead of `rename`.